tasks/bash.sh

```diff
@@ -1,106 +1,153 @@
-#!/usr/bin/env bash
-# The install_shell.sh implementation from the puppetlabs-puppet_agent module can use this
-# by passing positional argument with value "platform" or "release" to get only the platform
-# or version string only. 
-
-minor () {
-    minor="${*#*.}"
-    [ "$minor" == "$*" ] || echo "${minor%%.*}"
+#!/bin/bash
+
+# This script may be called outside of a task, e.g. by puppet_agent
+# so we have to just paste this code here.  *grumbles*
```

Comment:
Grumbles that we cant use a bash helper library in a bolt module to pull out some of these "standary" helper methods?

Reply:
It's more that the `puppet_agent `module invokes this script directly, like `bash.sh release".  So we can't keep this code in a separate helper file.  Hopefully that can change sometime...